hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRateRegistryTest.java

```diff
@@ -40,12 +40,12 @@ public class MaxRateRegistryTest extends ZookeeperBaseTest {
     private final ZookeeperPaths zookeeperPaths = new ZookeeperPaths("/hermes");
     private final ConfigFactory configFactory = new MutableConfigFactory();
     private final String consumerId = configFactory.getStringProperty(Configs.CONSUMER_WORKLOAD_NODE_ID);
-    private final String cluster = configFactory.getStringProperty(Configs.KAFKA_CLUSTER_NAME);
+    private final String cluster = "primary-dc";
```

Comment:
You can do something like `new KafkaProperties().getClusterName()`, thanks to this we wouldn't have to change this static value anymore.